addons/frag/functions/fnc_doFrag.sqf

```diff
@@ -0,0 +1,68 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Jaynus, NouberNou, Lambda.Tiger
+ * This function handles creating both random and targeted fragments as well
+ * as handling some of the performance optimizations.
+ *
+ * Arguments:
+ * 0: projectile that's fragmenting <OBJECT>
+ * 1: ASL position of projectile <ARRAY>
+ * 2: velocity of projectile <ARRAY>
+ * 3: projectile cfgAmmo classname <STRING>
+ * 4: getShotParents of projectile at EH <ARRAY>
```

Comment:
```suggestion
 * 0: Projectile that's fragmenting <OBJECT>
 * 1: ASL position of projectile <ARRAY>
 * 2: Velocity of projectile <ARRAY>
 * 3: Projectile cfgAmmo classname <STRING>
 * 4: getShotParents of projectile at EH <ARRAY>
```

Reply:
Done, also since the first argument isn't currently used, I'm removing it